libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -1191,6 +1192,19 @@ internal object MapboxRouteLineUtils {
         } ?: false
     }
 
+    internal fun resetCache() {
+        repeat(EXTRACT_ROUTE_DATA_CACHE_SIZE) { count ->
+            val route = DirectionsRoute.fromJson(
+                "{\"weight_name\":\"auto\",\"weight\":1.935,\"duration\":1.72," +
+                    "\"distance\":0.0,\"legs\":[{\"via_waypoints\":[]," +
+                    "\"admins\":[{\"iso_3166_1_alpha3\":\"USA\",\"iso_3166_1\":\"US\"}]," +
+                    "\"weight\":1.935,\"duration\":1.72,\"steps\":[],\"distance\":0.0," +
+                    "\"summary\":\"Irving Street\"}],\"geometry\":\"{routeGeometry$count\"}"
+            )
```

Comment:
What is the goal of this placeholder route?

Reply:
The key for the LRU cache is the function and its arguments.  The place holder route is an argument so they will replace any other existing items in the cache. Since there is no data in the route like steps etc. no calculations will be executed. So the net result is the routes in the cache get replaced with empty routes effectively leaving the cache empty.